### ISSUE TITLE
specifing new target bed

### ISSUE THREAD

**Issue by nitzankol** [Clinical-Genomics/MIP#1994]
I put a bed file called grch38_kit_name.bed in the reference_dir but I still get "ERROR Could not find intendended '.bed file ending' for target file '' . Is there a specific naming convention I need to follow?
I have also tried to provide the file using the --exome_target_bed parameter but the program complains that it provided only for the first sample. I tried target.bed=ID1; target.bed=ID2;target.bed=ID3 . How should I use it to provide the target for all samples?

**Comment by jemten** [comment]
Hello @nitzankol 👋 
It's a little cryptic but you can supply your own bed file using this command. Here sample 1 and 2 have one bed file and sample 3 another.
```Shell
--exome_target_bed Agilent_SureSelect.V5.GRCh37.70_targets.bed=Sample1,Sample2 \
--exome_target_bed Agilent_SureSelectFocusedExome.V1.GRCh37_targets.bed=Sample3 \
```

**Comment by nitzankol** [comment]
Hi Andres,
Thank you for your help.
I hope you can help with my next problem -
I get an error
[FATAL] 2022/07/07 06:04:07 MIP_ANALYSE - Could not find intended
vcfanno_config file:
TEST_REFERENCES!/grch38_loqusdb_snv_indel_-2020-03-24-.vcf.gz
I can't find where I need to define TEST_REFERENCES - I don't have it on my
config.yaml

thank you very much
Nitzan

‫בתאריך יום ב׳, 20 ביוני 2022 ב-12:57 מאת ‪Anders Jemt‬‏ <‪
***@***.***‬‏>:‬

**Comment by jemten** [comment]
answered/closed in #1997
